src/simulation/dynamics/spinningBodies/spinningBodiesOneDOF/spinningBodyOneDOFStateEffector.i

```diff
@@ -16,7 +16,17 @@
  OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 
  */
-
+#include <stdexcept>
+
+%exception { 
```

Comment:
I'm not following what this swig code is doing?  Could you please add some comments on what is being done here?